subprojects/language-native/src/main/java/org/gradle/language/swift/SwiftPlatform.java

```diff
@@ -26,4 +26,5 @@
  */
 @Incubating
 public interface SwiftPlatform extends NativePlatform {
+    SwiftVersion getSwiftVersion();
```

Comment:
Should add a comment to define what this means. Is it the source language version? The compiler version? The target ABI and standard library version? They're different things, even though they change together at the moment.

If it's the source language version, we should probably not include this on `SwiftPlatform` as `SwiftPlatform` represents the target platform not the source platform, and instead perhaps just expose this directly on `SwiftBinary`.